prime-router/docs/proposals/0020-univeral-pipeline-topics.md

```diff
@@ -0,0 +1,63 @@
+# Topic partitioning in the Universal Pipeline
+
+## Introduction
+
+The Universal Pipeline is meant to be the path for all data going through ReportStream. This includes Electronic
+Laboratory Reporting (ELR) as well as Electronic Test Orders and Results (ETOR), which are associated with different
+message types and have different purposes. These two use cases utilize different message types, require different data
+fields to be provided, and will likely be relevant to different sets of senders and receivers.
+
+## Goals
+
+Since ELR and ETOR have different purposes and their data is held to different expectations, we want organization
+settings to prevent the two data types (as well as future other data types) from getting mixed up between
+organizations (senders/receivers) that are intended for the other data type. The solution for this should be simple,
+foolproof, and independent as laid out here:
+
+- Simple: The Requirement is best described with an example: It is likely that ETOR onboarding and operations will be
+  managed by Technical Assistants (TA) hired directly by CDC, not by people on the ReportStream team. Those
```

Comment:
We should follow-up with ETOR on this to confirm this is the plan. RS does NOT want to be doing ETOR onboarding.

Reply:
Ok, yea I'll bring it up on a standup next week, if it's not the plan, or if it's not clear enough to speculate, I can come back and edit that bit out.